pkg/webhook/validator_result.go

```diff
@@ -63,6 +63,12 @@ type PolicyResult struct {
 	// This field is only available for evaluation if
 	// CIP.Spec.Policy.IncludeObjectMeta is set to true.
 	ObjectMeta interface{} `json:"metadata,omitempty"`
+
+	// TypeMeta contains the TypeMeta for the resource that was evaluated.
+	//
+	// This field is only available for evaluation if
+	// CIP.Spec.Policy.IncludeTypeMeta is set to true.
+	TypeMeta interface{} `json:"typemeta,omitempty"`
```

Comment:
is `typemeta` intentional or you want `typeMeta` similar to `authorityMatches` ?

Reply:
It was intentional to match metadata. Since TypeMeta is not serialized, well, it's inlined in k8s resources, figured this was ok.